datafusion/common/src/config.rs

```diff
@@ -1383,6 +1386,20 @@ impl ConfigField for TableParquetOptions {
         // Determine the key if it's a global or column-specific setting
         if key.contains("::") {
             self.column_specific_options.set(key, value)
+        } else if key.eq("metadata") {
+            for maybe_pair in value.split('_') {
```

Comment:
What does the `_` delimiter signify here? I didn't notice an example in the tests which included a `_` in the value.

Reply:
Nice catch. Also a good reason for why I needed to make the test cases more verbose to explain the syntax (and catch errors). Change pushed; thank you.